tests/integration_tests.rs

```diff
@@ -832,6 +832,42 @@ fn disable_pager_if_disable_paging_flag_comes_after_paging() {
         .stdout(predicate::eq("hello world\n").normalize());
 }
 
+#[test]
+fn disable_pager_if_pp_flag_comes_after_paging() {
+    bat()
+        .env("PAGER", "echo pager-output")
```

Comment:
(Looks like this has a semantic merge conflict with https://github.com/sharkdp/bat/pull/2602, so I'll wait with reviewing this PR until that PR is merged and this PR is rebased on top)

Reply:
It has now been merged, so can you rebase on the main branch please?